contributing-guides/style-guide.md

```diff
@@ -390,6 +390,42 @@ For example, `[d]ownload` in English may be translated into `[d]escargar` in Spa
 
 ## Example commands
 
+- The examples of pages in the subdirectory `common` for UNIX-like operating systems (e.g. `pages/common/tar.md`), preferably, should be written with the shell syntax specified by the [latest POSIX standard](https://pubs.opengroup.org/onlinepubs/9699919799/utilities/V3_chap02.html#tag_18).
```

Comment:
```suggestion
### POSIX-compliant syntax

- The examples of pages in the subdirectory `common` for UNIX-like operating systems (e.g. `pages/common/tar.md`), preferably, should be written with the shell syntax specified by the [latest POSIX standard](https://pubs.opengroup.org/onlinepubs/9699919799/utilities/V3_chap02.html#tag_18).
```

We generally add all contents behind a third-level heading in the style guide since the 2nd level heading is used for the main topics.

----

Also, I suggest moving this to the last part of this section to not confuse new users (who don't know about the POSIX standard).